src/status_im/contexts/chat/messenger/messages/list/view.cljs

```diff
@@ -185,8 +186,8 @@
                                 messages.constants/top-bar-height
                                 messages.constants/header-container-top-margin)
         background-color     (colors/theme-colors
-                              (colors/custom-color cover-bg-color 50 20)
-                              (colors/custom-color cover-bg-color 50 40)
+                              (colors/custom-color chat-color 50 20)
```

Comment:
Hi @seanstrom - I think we deprecated `custom-color`. instead if you can use `resolve-color method.
The end result should be 
```
      (colors/resolve-color chat-color theme 20)
       (colors/resolve-color chat-color theme 40)
```

Reply:
Sounds good will do 👍